include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -223,7 +226,11 @@ Parameters::Parameters()
     preconditioner_pressure_block(SchurComplementPreconditioner::PressureConvectionDiffusion),
     multigrid_data_pressure_block(MultigridData()),
     exact_inversion_of_laplace_operator(false),
-    solver_data_pressure_block(SolverData(1e4, 1.e-12, 1.e-6, 100))
+    solver_data_pressure_block(SolverData(1e4, 1.e-12, 1.e-6, 100)),
+
+    // Solving mass system (projection) if Matrix-free inverse mass operator is not avaliable.
```

Comment:
a comment like this should appear where the parameters are defined (PreconditionerMass for example). Here in the parameter file, such a detailed explanation is not needed in my opinion. I would rather write e.g. "only relevant for HDIV case" here